docs/en-US/component/popconfirm.md

```diff
@@ -55,7 +55,6 @@ popconfirm/trigger-event
 | ---------------------------------- | --------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- | -------------- |
 | title                              | Title                                                                                               | ^[string]                                                                    | —              |
 | effect ^(2.11.2)                   | Tooltip theme, built-in theme: `dark` / `light`                                                     | ^[enum]`'dark' \| 'light'` / ^[string]                                       | light          |
-| close-on-press-escape ^(2.11.2)    | whether the popConfirm can be closed by pressing ESC                                                | ^[boolean]                                                                   | true           |
```

Comment:
Should we keep it in the document and add a deprecated tag or put a statement at the beginning of the document?

Reply:
I do not think it is necessary to keep it. after all, it is a new addition and very few users use it.